controller/users.go

```diff
@@ -360,7 +360,7 @@ func isUsernameUnique(appl application.Application, username string, identity ac
 func isEmailUnique(appl application.Application, email string, user account.User) (bool, error) {
 	usersWithSameEmail, err := appl.Users().Query(account.UserFilterByEmail(email))
 	if err != nil {
-		log.Error(context.Background(), map[string]interface{}{
+		log.Error(nil, map[string]interface{}{
```

Comment:
I did consider this but then realized we don't really need the extra info which ctx gives us in this case :)